pkg/cloudprovider/aws/cloudprovider.go

```diff
@@ -157,11 +156,11 @@ func (c *CloudProvider) Validate(ctx context.Context, constraints *v1alpha5.Cons
 func (c *CloudProvider) Default(ctx context.Context, constraints *v1alpha5.Constraints) {
 	vendorConstraints, err := v1alpha1.Deserialize(constraints)
 	if err != nil {
-		logging.FromContext(ctx).Errorf("Failed to deserialize provider, %s", err.Error())
+		logging.FromContext(ctx).Fatalf("Failed to deserialize provider, %s", err.Error())
```

Comment:
I think we want to keep this as Errorf

Reply:
Yes, I think a bad rebase as I didn't touch this code.